content/bibs/pubs.bib

```diff
@@ -1480,20 +1480,19 @@ @article{shenLearningInterpretableCharacteristic2020
 	copyright = {All rights reserved}
 },
 
-@article{pandaNonparMANOVAIndependence2021,
-	title = {Nonpar MANOVA via Independence Testing},
+@misc{panda2023highdimensional,
+	title = {High-Dimensional and Universally Consistent K-Sample Tests},
 	author = {Panda, Sambit and Shen, Cencheng and Perry, Ronan and Zorn, Jelle and Lutz, Antoine and Priebe, Carey E. and Vogelstein, Joshua T.},
-	year = {2021},
 	author+an = {1=trainee;2=trainee;3=trainee;7=highlight},
-	keywords = {in-review},
-	url = {https://arxiv.org/abs/1910.08883},
-	month = {4},
-	journal = {arXiv},
-	abstract = {The k-sample testing problem tests whether or not k groups of data points are sampled from the same distribution. Multivariate analysis of variance (MANOVA) is currently the gold standard for k-sample testing but makes strong, often inappropriate, parametric assumptions. Moreover, independence testing and k-sample testing are tightly related, and there are many nonparametric multivariate independence tests with strong theoretical and empirical properties, including distance correlation (Dcorr) and Hilbert-Schmidt-Independence-Criterion (Hsic). We prove that universally consistent independence tests achieve universally consistent k-sample testing and that k-sample statistics like Energy and Maximum Mean Discrepancy (MMD) are exactly equivalent to Dcorr. Empirically evaluating these tests for k-sample scenarios demonstrates that these nonparametric independence tests typically outperform MANOVA, even for Gaussian distributed settings. Finally, we extend these non-parametric k-sample testing procedures to perform multiway and multilevel tests. Thus, we illustrate the existence of many theoretically motivated and empirically performant k-sample tests. A Python package with all independence and k-sample tests called hyppo is available from https://hyppo.neurodata.io/.},
+	year = {2023},
+	month = {10},
+	number = {arXiv:1910.08883},
 	eprint = {1910.08883},
-	archiveprefix = {arXiv},
 	primaryclass = {cs, stat},
-	eprinttype = {arxiv},
+	publisher = {{arXiv}},

```

Comment:
Is this supposed to be double brackets?

Reply:
@sampan501 Is the double bracket intentional? If not, can you fix then I will merge :)